apps/condo/domains/organization/integrations/sbbol/sync/index.js

```diff
@@ -87,7 +87,6 @@ const sync = async ({ keystone, userInfo, tokenSet, reqId }) => {
         ...dvSenderFields,
         name: userInfo.name || userInfo.OrgName,
         importId: userInfo.userGuid,
```

Comment:
Maybe to exclude both `importId` and `importRemoveSystem` from `userData` and introduce separate variable `externalIdentityData` to avoid confusion, because `importId` is still presented in `userData` and it seems like it is for `User` record.

Reply:
importId are field comes from sbbol adapters